examples/isostasy_calculation.py

```diff
@@ -0,0 +1,54 @@
+"""
+Calculation of
+"""
+import matplotlib.pyplot as plt
+import cartopy.crs as ccrs
+import harmonica as hm
+
+# Load the elevation model
+data = hm.datasets.fetch_topography_earth()
```

Comment:
It might be better to slice the data to get a single continent instead of doing the whole globe. Maybe Africa to avoid subduction zone? You can slice the `Dataset` using:

```
data_africa = data.sel(latitude=slice(lat_min, lat_max), longitude=slice(lon_min, lon_max))
```